docs/source/conf.py

```diff
@@ -14,10 +14,10 @@
 # -- Project information -----------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
-project = '<Name>'
+project = 'Logik'
 copyright = '2024, Zero ASIC'
 author = 'Zero ASIC'
-release = '0.0.1'
+release = '0.0.6'
```

Comment:
this will get out of sync very quickly, please see https://github.com/siliconcompiler/siliconcompiler/blob/f68bed60ceee3dbe7f83dd99d8c4dd6dbf50fbcf/docs/conf.py#L34